src/App.jsx

```diff
@@ -99,7 +104,7 @@ const App = () => {
 
     if (session.info.isLoggedIn) {
       localStorage.setItem('loggedIn', true);
-      fetchData();
+      setupPod();
```

Comment:
Does this function need to be called on every login? I feel like we can call this once, the first time a user enters PASS, setup any files we need, then never call it again.

Reply:
That sounds fine then